### ISSUE TITLE
Will community plugins be added to the main project?

### ISSUE THREAD

**Issue by AsafEitani** [volatilityfoundation/volatility3#864]
Hello, I was wondering why the community plugins - specifically those that win the volatility contest are not merged into the main branch.
Is there a problem of license or is the issue is that no one opened a PR on that.
I started writing plugins to volatility3 just to find that the same plugin (file list) was already developed and won last year contest.
Is there a reason?

I can open a PR merging those plugins.

**Comment by ikelos** [comment]
Hiya, I think the issue is more over who supports the plugins.  Some authors may wish to be able to update them quickly by keeping the plugins on their own github, otherwise may wish to add them to a community area, but then hope that they'll get updated by the core team when bugs arise (which puts an extra burden on the core team).  One way around this would be a community repo, much like the volatility 2 community repo, but so far we haven't had the time to develop the idea or turn it into something real.  I think git submodules may be a way of allowing developers to keep hosting the plugins themselves, whilst still allowing a list to be maintained (and possibly even branches for any versions).

So, in short, mostly it comes down to time rather than any legal issues, I believe.

**Comment by atcuno** [comment]
This seems like ikelos documented the avenues to core inclusion. Please re-open with further questions, if needed.
